README.md

```diff
@@ -53,6 +53,40 @@ There are some mandatory fields, each marked with "MANDATORY" in a comment.
 
 See [monitoring.md](docs/monitoring.md).
 
+### Attribute redaction
+
+While Kubernetes `v1.Secrets` are designed to separate secret fields into their
+own types, referenced in other types' fields such as container environment
+variable `secretKeyRefs`, some users may wish to prevent certain fields on
+certain types from being sent to Snyk, while still scanning other fields on that
+type.
+
+The scanner supports this sort of redaction, as an optional array of paths
+alongside group-version-kind scan config:
+
+```yaml
+- apiGroups: ["apps"]
+    versions: ["*"]
+    resources:
+      - replicasets
+      - daemonsets
+      - deployments
+      - statefulsets
+    redactions:
```

Comment:
There's no specific use case to highlight here just yet, so feel free to discard this feedback, but I could envision more options here in the future that live outside the path syntax itself. Should we start with this as an array of objects - potentially with a `path` key for each, rather than just strings?

The alternative is assuming we'd support a mix of strings and object configurations here in the future (there was some objection to this shared in another thread), or I guess we'd introduce some other configuration that superseded this one.

Reply:
Interesting, what specifically do you envision? I hadn't thought that we might want to scope by jsonpath-ish address as opposed to scoping by GVK.

I'm open to ideas but if we're not sure, I'd say probably YAGNI for now rather than introduce complexity to the config.